BookB-Pset_GeoScienceFeatureTypeGeologicalUnit (#625)
bSI-InfraRoom/IFC-Specification: IFC4x3/Properties/g/GeologicUnitType_3kvkEL4rr7TelKN6g8aSiF/Documentation.md

> The type of GeologicUnit (as defined in GeoSciML).

Reviewer: ```suggestion
>NOTE Definition according to GeoSciML: The type of GeologicUnit.
```
